gammapy/data/observations.py

```diff
@@ -342,6 +382,61 @@ def select_time(self, time_interval):
         obs.obs_filter = new_obs_filter
         return obs
 
+    @classmethod
+    def from_file(cls, event_file, irf_file):
+        """Create an Observation from a Event List and IRF file.
+        Ease the creation of a single Observation without creating an entire `DataStore`
+        that is without the need of HDU and OBS index files
+        """
+        log.info(f"reading the Event List and GTI from {event_file}")
+        event_file = make_path(event_file)
+        if not event_file.exists():
```

Comment:
Catching this error seems unnecessary. If the file does not exist `EventList.read()` should fail the same way.

Reply:
Ok, did not know that the `.read` would have failed, removed the check.